parlai/crowdsourcing/projects/humaneval/single_model_eval/hydra_configs/conf/example.yaml

```diff
@@ -0,0 +1,37 @@
+#@package _global_
+defaults:
+  - /mephisto/blueprint: model_chat_blueprint
+  - /mephisto/architect: local
+  - /mephisto/provider: mock
+mephisto:
+  blueprint:
+    annotations_config_path: ${task_dir}/task_config/annotations_config.json
+    onboarding_qualification: single_model_eval__onboarding
+    block_qualification: single_model_eval__${current_time}__block  # Use the current time in the block, like with Acute-Evals
+    chat_data_folder: ${task_dir}/model_chat/
+    model_opt_path: ${task_dir}/task_config/model_opts.yaml
+    num_turns: 6
+    task_model_parallel: true
+    check_acceptability: true
+    include_persona: true
+    conversation_start_mode: 'hi'
+    annotation_question: "Please answer the following:"
+    conversations_needed_string: "blender_90M:10"
+    final_rating_question: "Please rate how much you'd prefer to talk to your partner for a long conversation. (1: Would not at all prefer, 5: Would very much prefer)|Please rate how human your partner sounds. (1: Very inhuman, 5: Very human)|Please rate how interesting your partner is. (1: Very boring, 5: Very interesting)"
```

Comment:
Can we wrap this line?

Reply:
Yeah, good call - fixed this and then re-tested to make sure the questions still render properly